tests/unit/nupic/algorithms/knn_classifier_test.py

```diff
@@ -29,6 +29,34 @@
 
 class KNNClassifierTest(unittest.TestCase):
 
+ 
+  
+  
+  
+  def testSparsifyVector(self):
```

Comment:
Also, it isn't very obvious what each of the four are testing so make sure to add proper doc strings / explanatory names.